src/main/scala/com/avast/clients/stor/StorClient.scala

```diff
@@ -11,3 +15,9 @@ trait StorClient[F[_]] {
 
   def post(sha256: Sha256)(is: InputStream): F[Either[StorException, PostResult]]
 }
+
+object StorClient {
+  def fromConfig[F[_]: FromTask](config: Config): StorClient[F] = ???
```

Comment:
I would prefer to have built-in support for `Future`.

Reply:
That's a subject to discussion :-) But in general, me too.